spec/support/corpus.rb

```diff
@@ -10,10 +10,16 @@ module MutantSpec
   #
   # rubocop:disable MethodLength
   module Corpus
+    # Not in the docs. Number from chatting with their support.
+    # 2 processors allocated per container, 4 processes works well.
+    CIRCLE_CI_CONTAINER_PROCESSES = 4
+    TMP                           = ROOT.join('tmp').freeze
```

Comment:
Does the temporary folder need to be under the current root path? Why not just rely on something like `Dir.tmpdir`?